### PR TITLE
docs: rst 2 markdown

### PR DESCRIPTION
**Pull request by zramsay**
and other little fixes. Some of the docker commands in `tm-bench` are out of date (version 0.12.1), perhaps we want to push a `tendermint/tendermint:latest` docker image on release so we don't have to update the docs

also removed the `docker/` and `mintnet-kubernetes/` directories since these have move to `tendermint/tendermint/DOCKER` and `tendermint/tendermint/networks` respectively.

### CODE REVIEW

**Comment by greg-szabo** [comment]
I'm ok with the changes. I just want to point out that we're removing mintnet too with this PR. If the new `localnet` commands work fine and cover all requirements that mintnet provided, then this is good.

File: mintnet-kubernetes/app.template.yaml [hunk 1]
```diff
@@ -1,265 +0,0 @@
----
```
Comment (melekes) [inline mintnet-kubernetes/app.template.yaml]: I am against removing this.

**Comment by melekes** [comment]
I don't see respective PR with `mintnet-k8s` for tendermint

**Comment by zramsay** [comment]
my understanding is the the `make localnet-start` and associated commands _plus_ the terraform & ansible scripts have us fully covered in terms of tools for deploying chains locally or in the cloud

**Comment by greg-szabo** [comment]
Hm. Anton has a point: we don't have a Kubernetes/swarm/whatever solution in place to bring up a network in Docker containers remotely.
Gaia has remotenet (not dockerized but at least it gives you a full network in the cloud) but that's not available for tendermint either. (Except the sentrynet command which doesn't create validators.)

Let's put a big "not maintained any more" sign in the mintnet folder for now and keep it around. We'll deco it once we have better remotenet for tendermint.

**Comment by zramsay** [comment]
SGTM

**Pull request merged**